elescope.el

```diff
@@ -45,6 +45,14 @@ Defaults to 1 which makes all clones shallow clones."
   :group 'eslescope
   :type 'integer)
 
+
+(defcustom elescope-use-namespace nil
+  "Use project username as namespace for path.
+
+Default as nil"
+  :group 'byte-elescope
+  :type 'integer)
```

Comment:
Should probably be a boolean? Also `byte-elescope`? Wouldn't this be `elescope`?

Reply:
Mistake🙈